staging/src/k8s.io/cri-api/pkg/apis/testing/fake_runtime_service.go

```diff
@@ -194,6 +194,11 @@ func (r *FakeRuntimeService) RunPodSandbox(config *runtimeapi.PodSandboxConfig,
 			Network: &runtimeapi.PodSandboxNetworkStatus{
 				Ip: FakePodSandboxIPs[0],
 			},
+			Linux: &runtimeapi.LinuxPodSandboxStatus{
```

Comment:
[check](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kuberuntime/kuberuntime_manager.go#L445) will always restart sandbox without this

Reply:
Please add a comment about that then :)